src/docfx/cli/Docfx.cs

```diff
@@ -96,7 +99,7 @@ internal static async Task<int> Run(string[] args)
                 {
                     Log.Write(dex);
                     errorLog.Write(dex.Error, isException: true);
-                    return 1;
+                    return 0;
```

Comment:
The publish.json will still mark error item as has_error, then the change should be safe.